HACKING.adoc

```diff
@@ -511,9 +511,12 @@ than trying to open the incompatible artefacts produced by a Makefile build. In
 particular, you need to repeat the dune build every time you change the interface
 of some compilation unit, so that merlin is aware of the new interface.
 
-You only need to run `configure` once, but you will need to run `make clean`
-every time you want to run `dune` after you built something with `make`;
-otherwise dune will complain that build artefacts are present among the sources.
+You only need to run `configure` once, but you will need to run `make
+clean-for-dune` every time you want to run `dune` after you built
```

Comment:
For some reason the GitHub renderer preserves [hard line breaks](https://docs.asciidoctor.org/asciidoc/latest/blocks/hard-line-breaks/) in [monospace spans](https://docs.asciidoctor.org/asciidoc/latest/text/monospace/). Maybe it interprets them similarly to [literal monospace spans](https://docs.asciidoctor.org/asciidoc/latest/text/literal-monospace/).

<img width="889" height="74" alt="Image" src="https://github.com/user-attachments/assets/22cab974-c9c9-47a5-8205-df1f61353b28" />

I can fix this in #14392.